en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md

```diff
@@ -495,17 +496,37 @@ LaTeX is installed, a beautifully formatted PDF file can be created
 using the same command structure:
 
 ```
-$ pandoc -o main.pdf main.md
+$ pandoc main.md -o main.pdf
+```
+
+If your document is written in languages other than English, you will likely need to use
+the XeLaTeX engine instead of plain LaTeX for .pdf conversion:
+
+```
+pandoc main.md --pdf-engine=xelatex -o main.pdf
 ```
 
-With time, you will be able to fine tune the formatting of PDF documents
-by specifying a LaTeX style file (saved to the same directory), and
-running something like:
+Make sure your tex editor supports the UTF-8 encoding. When using XeLaTeX for conversion into
+.pdf, instead of the `fontfamily` attribute in YAML, specify the `mainfont` attribute, to
```

Comment:
Explain to the reader why it's necessary to change the `fontfamily` attribute.

Reply:
I wish I knew. It seems to be an arbitrary decision and a source of much confusion in support forums. Nothing that I saw in the documentation explains the reason for this mechanic.

Reply:
ah sorry, I didn't necessarily mean that we needed a deep explanation 😂  - just add a word or two so that it's clear "when using XeLaTeX for conversion into .pdf, **you must** use `mainfont` instead of `fontfamily`"